WebInstall/explainer-bd.md

```diff
@@ -0,0 +1,358 @@
+
+**Background Document Installations**
+
+Installation of the background document is when the install API is invoked to install a web application different from the navigation context the API is called from. The app that will be installed can be in the same or a different origin. This enables the following use cases (among others):
+
+### **Suite of Web Apps**
+
+Building on the previous use case, the website can also provide a way to directly acquire other applications it might offer, like a dedicated "kids" version of the app, or a "sports" version of the app. Another example would be a family of software applications, like a productivity or photography suite, where each application is accessed from a different web page. The developer is in control and can effectively advertise and control their applications, without having to redirect users to platform-specific proprietary repositories, which is what happens now.
+
+### **SERP app install**
+
+Developers of Search Engines could use the API to include a way to directly install an origin that is an application. A new feature could be offered by search engines that could see them facilitating a frictionless way to acquire an app that a user is searching for. This could also aid discovery as a user might not be aware that a specific origin has an associated web application they could acquire. 
+
+### **Creation of online catalogs**
+
+ Another potential use case for the API is related to the creation of online catalogs. A web site/app can list and install web apps. A unique aspect of this approach is that since the applications that are installed are web apps, this could enable a new set of true cross-device, cross-platform app repositories. 
+ 
+  ![Install flow from an app repository](./apprepositoryinstallation.png)
+
+## Proposed Solution
+
+### The `navigator.install` method
+
+To install a web app, a web site would use the promise-based method `navigator.install([<install_url>[, <manifest_id>[, <params>]]]);`. This method will:
+
+* Resolve when an installation was completed.
+    * The success value will be an object that contains:
+        *  `id`: string with the processed `manifest_id` of the installed app.
+* Be rejected if the app installation did not complete. It'll reject with a [`DOMException`](https://developer.mozilla.org/en-US/docs/Web/API/DOMException) value of:
+    * `AbortError`: The installation/UX/permission (prompt) was closed/cancelled.
+    * `DataError`: There is no manifest file present, there is no `id` field defined in the manifest or there is a mismatch between the `id` passed as a parameter and the processed `id`.
+
+#### **Signatures of the `install` method**
```

Comment:
Should we consider calling out what's the same as the current document web-install vs. new? The intent here is to avoid rehashing the same conversation.

Reply:
I thought about that, as there is plenty of repetition, nonetheless I think for context it is better to have both explainers being meaty for now. I also plan to link to them accordingly once the background document one is in WICG.